src/index.tsx

```diff
@@ -36,12 +36,10 @@ const root = document.getElementById('root')
 if (
     process.env.NODE_ENV === 'production' &&
     window._env_ &&
-    (window._env_.SENTRY_ERROR_ENABLED || window._env_.SENTRY_PERFORMANCE_ENABLED)
```

Comment:
Is removal of SENTRY_PERFORMANCE_ENABLED expected?

Reply:
We are not removing SENTRY_PERFORMANCE_ENABLED we are just putting it behind SENTRY_ERROR_ENABLED. Because we can't send the performance only, Sentry init will send errors by default. So removed the check to make it clear that you can on and off performance but with errors

Reply:
Got it